lib/iris/tests/unit/fileformats/grib/load_convert/test_vertical_coords.py

```diff
@@ -22,7 +22,7 @@
 from __future__ import (absolute_import, division, print_function)
 from six.moves import (filter, input, map, range, zip)  # noqa
 
-# import iris tests first so that some things can be initialised
```

Comment:
Quite a few files have ìris tests` rather than ìris.tests`. I only corrected them to ìris.tests`(which I believe is correct) when I was already changing something to that file else this PR would be HUGE.